api/api-pipeline-selection-v1/src/test/groovy/com/thoughtworks/go/apiv1/pipelineselection/representers/PipelineSelectionsRepresenterTest.groovy

```diff
@@ -55,7 +55,6 @@ class PipelineSelectionsRepresenterTest {
 
       JsonFluentAssert.assertThatJson(actualJson).isEqualTo([
         pipelines: [
-          grp1: [],
```

Comment:
It's not obvious looking at the test that empty groups should not be rendered. I'd probably write an explicit test which makes the behavior explicit.

Reply:
Added separate testcase for empty pipeline groups.